README.rst

```diff
@@ -372,6 +372,19 @@ Port used by the remote syslog server.
 Facility for the syslog message if not provided by the entry being relayed.
 (see RFC5424 for list of facilities.)
 
+``default_severity`` (default ``6``)
+
+Severity for the syslog message if not provided by the entry being relayed.
+(see RFC5424 for list of priorities.)
+
+``format`` (default ``rfc5424``)
+
+Log format to use. Can be rfc3164, rfc5424 or custom.
+
+``logline`` (default ``null``)
+
+Custom logline format (ignored unless format is set to custom).
```

Comment:
An example of what it looks like or some other info on how to use this would be good.

Reply:
added an example and list of supported tags